java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/ClientManager.java

```diff
@@ -1,42 +1,35 @@
 package org.sagebionetworks.bridge.sdk;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
 import org.sagebionetworks.bridge.sdk.rest.ApiClientProvider;
 import org.sagebionetworks.bridge.sdk.rest.model.SignIn;
 
-import com.google.common.base.Preconditions;
-
 public class ClientManager {
 
-    private Config config = new Config();
+    private final Config config = new Config();
 
     private ClientInfo clientInfo = new ClientInfo.Builder().build();
     
-    /**
-     * This is the default, to look in the bridge-sdk.properties file, but it can 
-     * be overwritten when creating the manager.
-     */
-    private SignIn signIn = config.getAccountSignIn();
-
-    protected ApiClientProvider apiClientProvider;
+    private ApiClientProvider apiClientProvider;
```

Comment:
So you can change the ClientInfo first and have that change reflected in the ApiClientProvider that gets created, because I assume you need to do that.

This class needs to keep changing, I'm not sure yet what it needs to do. You might eventually initialize it with ClientInfo and a SignIn, I'm not sure.